### PR TITLE
feat: add flag depencency proto to rule

### PR DESCRIPTION
**Pull request by kentakozuka**
part of https://github.com/bucketeer-io/bucketeer/issues/963

### CODE REVIEW

File: proto/feature/clause.proto [hunk 1]
```diff
@@ -30,6 +30,8 @@ message Clause {
     LESS_OR_EQUAL = 8;
     BEFORE = 9;
     AFTER = 10;
+    // Attribute is feture ID and value is variation ID.
```
Comment (cre8ivejp) [inline proto/feature/clause.proto:33]:
```suggestion
    // Attribute is feature ID, and value is variation ID.
```

**Comment by kentakozuka** [comment]
@cre8ivejp Fixed 👍

**Comment by kentakozuka** [comment]
resolved conflicts.

**Review by cre8ivejp — APPROVED** [review]
Thank you!

**Pull request merged**